pkg/controllers/managementuser/rbac/podsecuritypolicy/binding.go

```diff
@@ -131,6 +132,13 @@ func (l *lifecycle) createPolicy(obj *v3.PodSecurityPolicyTemplateProjectBinding
 	objectMeta.Annotations[podSecurityPolicyTemplateParentAnnotation] = template.Name
 	objectMeta.Annotations[podSecurityPolicyTemplateVersionAnnotation] = template.ResourceVersion
 
+	// Setting annotations that doesn't contains podSecurityPolicyTemplateFilterAnnotation
+	for k, v := range template.Annotations {
```

Comment:
My two cents would be that we don't override any existing annotation that were set in the code. So:
 ```suggestion
	for k, v := range template.Annotations {
	  if _, ok := objectMeta.Annotations[k]; ok {
	    continue
	  }
	  objectMeta.Annotations[k] = v
```

Reply:
Good point @StrongMonkey, that's why just annotations that doesn't contains `.cattle.io/` are copied. Just "user" annotations should be overwritten with the value coming from psp template. 

Also, we don't want to copy annotations prefixed with`clusterscoped.controller.cattle.io/cluster-pspt-sync_` nor  `lifecycle.cattle.io/create.cluster-pspt-sync_` as they are maintained with the template `cleanPSPT` function, but with your proposal they would be copied.